assets/helpers/contributions.js

```diff
@@ -122,3 +130,25 @@ export function errorMessage(
   }
 
 }
+
+function contribCamelCase(contrib: Contrib) {
+
+  switch (contrib) {
+    case 'ANNUAL': return 'annual';
```

Comment:
Probably `case 'ANNUAL': return contrib.toLowerCase();` is better and we avoid many hard-coded strings.

Reply:
Hmm, I can't do that for `oneOff` and I feel it's perhaps a little confusing to have a function called `contribCamelCase` returning `toLowerCase`?